app/src/main/java/com/schibsted/tapdadev/MainPresenter.java

```diff
@@ -68,6 +72,11 @@ private void show(int developer) {
     private void show(View view) {
```

Comment:
What do you think about changing this name's method? Actually, if the responsability of the method ``show`` is to show a developer... why it is disappearing later? I think that simply having a great method name would be great.